3rdparty/libtiff/tif_fax3.c

```diff
@@ -840,7 +840,7 @@ static	int32 find1span(unsigned char*, int32, int32);
  * table.  The ``base'' of the bit string is supplied
  * along with the start+end bit indices.
  */
-inline static int32
+__inline static int32
```

Comment:
Could you fix it [here](https://github.com/opencv/opencv/blob/9de87d96ccfa2f0aeb2ea74e110565e6dfc3f047/3rdparty/libtiff/CMakeLists.txt#L39) ?

Reply:
Thanks for your reply.  Let me do it.
You always surprise me by showing a better solution!